modules/tinymce/src/plugins/image/demo/ts/demo/Demo.ts

```diff
@@ -26,7 +26,7 @@ tinymce.init({
     console.log(blobInfo);
     Delay.setTimeout(function () {
       success('https://www.google.com/logos/google.jpg');
-    }, 5000);
+    }, 50);
```

Comment:
I'm not sure if this demo has 5000 for a specific test case, but 5000 better simulates an upload time

Reply:
ooopssss, had that for test and forgot to roll back... Thanks